tests/test_session.py

```diff
@@ -250,6 +250,12 @@ def test_auth_can_be_passed_as_basic_header(session_setup: Tuple[MagicMock, Sess
     assert isinstance(session.auth, kinto_http.BearerTokenAuth)
     assert session.auth.type == "Bearer+OIDC"
     assert session.auth.token == "abcdef"
+    session = create_session(auth="Basic asdfghjkl;")
```

Comment:
I know that this part very related, but we try to maintain a test per scope. Could you please split this test into a dedicated one?